internal/logic_handlers/organizationlogic/service_test.go

```diff
@@ -0,0 +1,674 @@
+					Persistor:  &organizationlogicfakes.FakePersistor{},
+					TxProvider: &mockTxProvider,
+					Logger:     mockLogger,
+					Cleanup:    cleanup,
+				}, cleanup
+			},
+			mutations: func(t *testing.T, db *sqlx.DB) {
+
+			},
+			assertions: func(t *testing.T, params *model.UpdateOrganization, organization *model.Organization, err error) {
+				assert.Error(t, err, "unexpected error")
+				assert.Nil(t, organization, "unexpected nil organization")
+			},
+		},
+		{
+			name: "fail-internal-server-error",
+			args: argsUpdateOrganizations{
+				ctx: context.TODO(),
+				params: &model.UpdateOrganization{
+					Id: 1,
+					UserId: null.Int{
+						Int:   3,
+						Valid: true,
+					},
+				},
+			},
+			getDependencies: getConcreteDependencies,
+			mutations: func(t *testing.T, db *sqlx.DB) {
+				testhelper.DropTable(t, db, mysqlmodel.TableNames.Organization)
+			},
+			assertions: func(t *testing.T, params *model.UpdateOrganization, organization *model.Organization, err error) {
+				assert.Error(t, err, "unexpected error")
+				assert.Nil(t, organization, "unexpected nil organization")
+			},
+		},
+	}
+}
+
+func TestService_UpdateOrganization(t *testing.T) {
+	for _, tt := range getTestCasesUpdateOrganizations() {
+		t.Run(tt.name, func(t *testing.T) {
+			_dependencies, cleanup := tt.getDependencies(t)
+			defer cleanup()
+
+			svc, err := New(&Config{
+				TxProvider: _dependencies.TxProvider,
+				Logger:     _dependencies.Logger,
+				Persistor:  _dependencies.Persistor,
+			})
+			require.NoError(t, err, "unexpected new error")
+
+			tt.mutations(t, _dependencies.Db)
+
+			organization, err := svc.UpdateOrganization(tt.args.ctx, tt.args.params)
+			tt.assertions(t, tt.args.params, organization, err)
+		})
+	}
+}
+
+type argsCreateOrganizations struct {
+	ctx          context.Context
+	organization *model.CreateOrganization
+}
+
+type testCaseCreateOrganization struct {
+	name            string
+	getDependencies func(t *testing.T) (*dependencies, func(ignoreErrors ...bool))
+	args            argsCreateOrganizations
+	mutations       func(t *testing.T, db *sqlx.DB)
+	assertions      func(t *testing.T, organization *model.Organization, err error)
+}
+
+func getTestCasesCreateOrganization() []testCaseCreateOrganization {
+	return []testCaseCreateOrganization{
+		{
+			name:            "success",
+			getDependencies: getConcreteDependencies,
+			args: argsCreateOrganizations{
+				ctx: context.TODO(),
+				organization: &model.CreateOrganization{
+					Name:   "Demby",
+					UserId: 1,
+				},
+			},
+			assertions: func(t *testing.T, organization *model.Organization, err error) {
+				require.NoError(t, err, "unexpected error")
+				require.NotNil(t, organization, "unexpected nil organization")
+
+				require.NotEqual(t, 0, organization.Id, "unexpected nil organization")
+				require.NotEmpty(t, organization.Name, "unexpected empty organization name")
+				require.NotEqual(t, 0, organization.CreatedBy, "unexpected empty Created_by")
```

Comment:
again better to be specific on the check.. like check the name is demby, and check that the id is whatever was returned in the mutation